src/Hangfire.LiteDB/LiteDbMonitoringApi.cs

```diff
@@ -105,7 +107,7 @@ public JobDetailsDto JobDetails(string jobId)
                     CreatedAt = x.CreatedAt,
                     Reason = x.Reason,
                     Data = x.Data
-                }).ToList();
+                }).OrderBy(x => x.CreatedAt).ToList();
```

Comment:
Should it be `OrderByDescending` ?

Reply:
Ready

Reply:
Example:
![image](https://user-images.githubusercontent.com/1153361/62801386-9ef0bc00-bab3-11e9-8760-2dab67c22c46.png)

Reply:
Looks great!